spec/features/idv/in_person_spec.rb

```diff
@@ -270,55 +270,84 @@
       allow(FeatureManagement).to receive(:reveal_gpo_code?).and_return(true)
     end
 
-    it 'requires address verification before showing instructions', allow_browser_log: true do
-      sign_in_and_2fa_user
-      begin_in_person_proofing
-      complete_all_in_person_proofing_steps
-      click_on t('idv.troubleshooting.options.verify_by_mail')
-      expect_in_person_gpo_step_indicator_current_step(
-        t('step_indicator.flows.idv.verify_address'),
-      )
-      click_on t('idv.buttons.mail.send')
-      expect_in_person_gpo_step_indicator_current_step(t('step_indicator.flows.idv.verify_address'))
-      complete_enter_password_step
-
-      expect_in_person_gpo_step_indicator_current_step(t('step_indicator.flows.idv.verify_address'))
-      expect(page).to have_content(t('idv.titles.come_back_later'))
-      expect(page).to have_current_path(idv_letter_enqueued_path)
-
-      click_idv_continue
-      expect(page).to have_current_path(account_path)
-      expect(page).not_to have_content(t('headings.account.verified_account'))
-      click_on t('account.index.verification.reactivate_button')
-      expect_in_person_gpo_step_indicator_current_step(t('step_indicator.flows.idv.verify_address'))
-      click_button t('idv.gpo.form.submit')
-
-      # personal key
-      expect_in_person_gpo_step_indicator_current_step(t('step_indicator.flows.idv.secure_account'))
-      expect(page).to have_content(t('titles.idv.personal_key'))
-      acknowledge_and_confirm_personal_key
+    context 'verify by mail not allowed for biometric' do
+      before do
+        allow(IdentityConfig.store).to receive(:no_verify_by_mail_for_biometric_comparison_enabled).
+          and_return(true)
+      end
 
-      expect(page).to have_current_path(idv_in_person_ready_to_verify_path)
-      expect_in_person_gpo_step_indicator_current_step(
-        t('step_indicator.flows.idv.go_to_the_post_office'),
-      )
-      expect(page).not_to have_content(t('account.index.verification.success'))
+      it 'does not present gpo as an option', allow_browser_log: true do
+        sign_in_and_2fa_user
+        begin_in_person_proofing
+        complete_all_in_person_proofing_steps
+        expect(page).not_to have_content(t('idv.troubleshooting.options.verify_by_mail'))
```

Comment:
What do you think about adding something to confirm we are on /verify/phone?  
ie: expect(page).to have_content(t('titles.idv.phone'))

If we were on the wrong view (if the steps got goofed up) this would pass. (If the steps got goofed I am sure a million other specs would fail but think it would be nice.)  I did confirm with binding.pry and SHOW_BROWSER that this is the correct view.

Reply:
Makes sense to me, I'll add